config/clusters/leap/daskhub-common.values.yaml

```diff
@@ -91,6 +91,7 @@ basehub:
           allowed_organizations:
             - leap-stc:leap-pangeo-base-access
             - leap-stc:leap-pangeo-full-access
+            - leap-stc:data-and-compute
```

Comment:
Unsure if this is actually needed?

Reply:
This is needed, the later parts are only filtering whats presented - but this is the part that makes them able to login at all.